### ISSUE TITLE
Hand of Justice lvl 80 proc rate too high

### ISSUE THREAD

**Issue by punims** [dalaranwow/dalaran-wow#5557]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Hand of justice trinket is proccing WAY too much, no proof added but if you equip it its clear the proc rate is around 18% while it doesnt downscale for level 80s (for a level 80 it should proc at about a 0.65% rate). Tired of seeing paladins abusing this in pvp

**Expected behaviour**: Tell us what should happen instead
mentioned above

**Steps to reproduce the problem**:

1. equip the trinket
2. see im right
3. idk

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
I did some testing of my own. It only took me this long because the trinket would not drop. 

1600 attacks and 40 procs in total. I also tested special abilities and it does not proc for those. How paladins abuse this in PvP is something I don't know, so if you can shed some light it will be appreciated.

![hand of justice](https://user-images.githubusercontent.com/24550199/43519524-725c9fd8-9598-11e8-9f47-a523b7efa917.png)

**Comment by Chenocide** [comment]
I had the same result when I tested it as feral. Seems to me it's paladin exclusive. The trinket is also used in PVE, not just PVP.

**Comment by lkrocek** [comment]
https://wowgaming.altervista.org/aowow/?item=11815

**Comment by Edzero** [comment]
It procs on seal of command which is why it's better for paladins.

It can proc from pretty much any melee "ability" including tiny abomination in a jar procs, bryntroll procs and I believe the damage proc from shadowmourne.

So it will stack considerably on a paladin with those items equipped.

**Comment by mrnuggets** [comment]
Please Loth, either fix this item, or remove it entirely from your server.

It's really annoying facing lvl 79 twinks using it, and not having a chance, cheers.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the reports. It should be already fine and applied on Algalon.

**Comment by mrnuggets** [comment]
Can someone test and confirm this? Doesn't seem to be live yet.

**Comment by Chenocide** [comment]
Might need a server reset first.

**Comment by fotlogo** [comment]
yes it seems 'fixed': i get 0 procs at hundreds of hits so i guess this is good enough
